_data/guides.yaml

```diff
@@ -85,4 +85,10 @@ categories:
       - title: Integrating with a PostgreSQL database
         url: /guides/database-integrating-with-postgresql
         description: Learn how to configure a datasource to connect to a PostgreSQL database.
+  - category: Testing
+    cat-id: testing
+    guides:
+      - title: Testing with Arquillian and JUnit 5
+        url: /guides/testing-arquillian-junit5
+        description: Learn how to write JUnit 5 tests using Arquillian for Jakarta EE based applications.
```

Comment:
Instead of Testing, at the OKR 3 Meeting, we proposed to get this guide under "Automation" to avoid having too many categories, ref: https://docs.google.com/document/d/1JoxoqN9Z5adpdNkTrWPdjLN3BroB4cBmV2ABYs-lm2I/edit?tab=t.0

If we prefer to have a specific category for Testing, that's ok too, so I just only mention it here for the record

Reply:
Automation seems fine. I'll make the change.